lib/serverless/aws-lambda.js

```diff
@@ -122,13 +126,26 @@ class AwsLambda {
       this.wrapFatal()
     }
 
-    return shim.bindCreateTransaction(wrappedHandler, new specs.TransactionSpec({ type: shim.BG }))
+    const nrHandler = shim.bindCreateTransaction(wrappedHandler, new specs.TransactionSpec({ type: shim.BG }))
+
+    // Response-streaming handlers are identified by symbol properties on the function.
+    // We propagate any symbols if they're present, so that the handler keeps its signature
+    // for any AWS features that rely on symbols
+    for (const symbol of Object.getOwnPropertySymbols(handler)) {
+      logger.trace(`Setting symbol ${symbol.toString()} on handler`)
+      nrHandler[symbol] = handler[symbol]
+    }
+
+    return nrHandler
 
     function wrappedHandler() {
       const args = shim.argsToArray.apply(shim, arguments)
+      logger.trace('handler args', args)
 
       const event = args[0]
-      const context = args[1]
+      logger.trace('Is this a response streaming Lambda function?', isStreamHandler)
```

Comment:
do we want this to ship?

Reply:
This could be useful as we roll out this feature. We're doing essentially the same thing for API Gateway, so it fits the pattern of trace logging for this module.